packages/adapter-vercel/image-loader.js

```diff
@@ -0,0 +1,49 @@
+// https://vercel.com/docs/concepts/image-optimization
+
+/**
+ * @param {string} src
+ * @param {number} width
+ * @param {{ quality?: number }} [options]
+ */
+export default function loader(src, width, options) {
```

Comment:
it'd probably make sense to use the type defined in the `.d.ts` file to keep them from going out of sync